test/es6/rlexe.xml

```diff
@@ -689,6 +689,18 @@
   </test>
   <test>
     <default>
+      <files>default-splitscope.js</files>
+      <compile-flags> -ES6DefaultArgs -ES6Generators -args summary -endargs</compile-flags>
+    </default>
+  </test>
+  <test>
+    <default>
+      <files>default-splitscope.js</files>
+      <compile-flags> -ES6DefaultArgs -ES6Generators -nodeferparse -args summary -endargs</compile-flags>
```

Comment:
Instead of nodeferparse  you should add ForceDeferParse - because that is where most of the issues have found.